admin/app/com/lucidchart/piezo/admin/controllers/TriggerFormHelper.scala

```diff
@@ -22,7 +22,7 @@ class TriggerFormHelper(scheduler: Scheduler) extends JobDataHelper {
 
   private def simpleScheduleFormUnapply(simple: SimpleScheduleBuilder) = {
     val simpleTrigger = simple.build().asInstanceOf[SimpleTrigger]
-    Some((simpleTrigger.getRepeatCount, simpleTrigger.getRepeatInterval.toInt))
+    Some((simpleTrigger.getRepeatCount, simpleTrigger.getRepeatInterval.toInt / 1000))
```

Comment:
This fixed an existing bug where editing a simple trigger caused the repeat interval to multiply by 1000 because it did not convert from milliseconds back to seconds.